### PR TITLE
Updating Readme file for the QA tested scenarios. Added ARIA, SEED and SHA224 enums.

### PR DESCRIPTION
**Pull request by captain-rohit**

### CODE REVIEW

File: examples/crypto_ops/main.go [hunk 1]
```diff
@@ -14,9 +14,9 @@ import (
 )
 
 const (
-	myAPIEndpoint string = "https://sdkms.fortanix.com"
-	myAPIKey string = "N2MwYThlYjgtMGZkNS00OWIxLWFkOWUt..."
-	keyName  string = "AES Key"
+	myAPIEndpoint string = "https://sit.smartkey.io"
+	myAPIKey      string = "NzI0OGM3MjgtNDA2Yy00MDkxLTk1NWMtNjE5ZDYwMDY1NTFhOmpNZmU2LW9xWjlMdEVIQ0lLQTg2UnUxWHJ4dXVEWWw4MjRBQlFpRmpyR01OS3pwSDdTTEF6RUZfb25kZVk5enJOQkhhNi1ETTktaUlfTHlpc1FMZkpn"
```
Comment (mzohreva) [inline examples/crypto_ops/main.go]: Please don't put secrets in a public repository

File: README.md [hunk 2]
```diff
@@ -40,3 +40,9 @@ this project or the open source license(s) involved.
 
 This project is primarily distributed under the terms of the Mozilla Public License (MPL) 2.0, see [LICENSE](./LICENSE) for details.
 Examples are subject to the Apache License 2.0, see [examples/LICENSE](./examples/LICENSE) for details.
+
+## Following are the scenarios tested
```
Comment (mzohreva) [inline README.md]: This is not the right place to put this information.

**Review by mzohreva — CHANGES_REQUESTED** [review]

**Review by govardhanfortanix — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (captain-rohit) [reply README.md]: @mzohreva removed it.

**Pull request merged**